datahub-frontend/app/react/auth/OidcResponseErrorHandler.java

```diff
@@ -0,0 +1,52 @@
+package react.auth;
+
+import org.pac4j.play.PlayWebContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import play.mvc.Result;
+
+import static play.mvc.Results.internalServerError;
+import static play.mvc.Results.unauthorized;
+
+
+public class OidcResponseErrorHandler {
+    private static final Logger _logger = LoggerFactory.getLogger("OidcResponseErrorHandler");
+
+    private static final String ERROR_FIELD_NAME = "error";
+    private static final String ERROR_DESCRIPTION_FIELD_NAME = "error_description";
+
+    public static Result handleError(final PlayWebContext context) {
+
+        _logger.warn("OIDC responded with an error: '{}'. Error description: '{}'",
+                getError(context),
+                getErrorDescription(context));
+
+        if (getError(context).equals("access_denied")) {
```

Comment:
Much better than the Play error page. Does this render just the static text on the screen?

Reply:
Yes, just static text. Thought that it would be sufficient and more light weight